pkg/types/powervs/platform.go

```diff
@@ -0,0 +1,63 @@
+package powervs
+
+// Platform stores all the global configuration that all machinesets
+// use.
+// Note: The subsequent mentions of future-TF support refer to work that is
+// undergoing and should be available to test well in time for 4.10 feature-
+// freeze. We do not plan to GA with these as required inputs.
+type Platform struct {
+
+	// ServiceInstanceID is the ID of the Power IAAS instance created from the IBM Cloud Catalog
+	ServiceInstanceID string `json:"serviceInstance"`
+
+	// PowerVSResourceGroup is the resource group for creating Power VS resources.
+	PowerVSResourceGroup string `json:"powervsResourceGroup"`
+
+	// Region specifies the IBM Cloud region where the cluster will be created.
+	Region string `json:"region"`
+
+	// Zone specifies the IBM Cloud colo region where the cluster will be created.
```

Comment:
Do we not want to spread the cluster across multiple zones in the region, where possible?

Reply:
it's not possible at this time without setting up additional tunnelling. should i note that here or somewhere else?